### DISCUSSION TITLE
Unable to install talib

### DISCUSSION THREAD

**Discussion by nedimmemic** [community/community#81233] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After reinstalling Python to 3.11. ver,  I have tried multiple times to install TA-Lib for 64 bit. Commands !pip install talib, nor conda install TA-Lib, do not do it. Please show me how to install ta-lib. My current installation of talib is in C:
C:Users\Name\anaconda3\site_packages.

nedim.memic21@gmail.com

**Comment by PlanetDestroyyer** [comment]
solution 1) Check us ENV, check if u r in right env else just print conda list or pip list

solution 2)
You could try the "[Unofficial Windows Binaries for Python Extension Packages](http://www.lfd.uci.edu/%7Egohlke/pythonlibs/#ta-lib) .

[http://www.lfd.uci.edu/~gohlke/pythonlibs/#ta-lib](http://www.lfd.uci.edu/%7Egohlke/pythonlibs/#ta-lib)

You probably need TA_Lib‑0.4.10‑cp35‑cp35m‑win_amd64.whl

solution 3)
follow [this](https://blog.quantinsti.com/install-ta-lib-python/)
